### PR TITLE
Ran pre-commit, added licenses

### PR DESCRIPTION
**Pull request by dherrada**

### CODE REVIEW

File: adafruit_azureiot/__init__.py [hunk 1]
```diff
@@ -1,24 +1,7 @@
-# The MIT License (MIT)
+# SPDX-FileCopyrightText: 2020 Jim Bennet for Adafruit Industries
```
Comment (FoamyGuy) [inline adafruit_azureiot/__init__.py:1]: There is an additional author `Elena Horton` listed below

File: adafruit_azureiot/constants.py [hunk 2]
```diff
@@ -1,24 +1,7 @@
-# The MIT License (MIT)
+# SPDX-FileCopyrightText: 2020 Jim Bennet for Adafruit Industries
```
Comment (FoamyGuy) [inline adafruit_azureiot/constants.py:1]: additional author `Elena Horton` is listed below

File: adafruit_azureiot/iot_error.py [hunk 3]
```diff
@@ -1,24 +1,7 @@
-# The MIT License (MIT)
+# SPDX-FileCopyrightText: 2020 Jim Bennet for Adafruit Industries
```
Comment (FoamyGuy) [inline adafruit_azureiot/iot_error.py:1]: additional author `Elena Horton` listed below. There are some more of these as well, I wont comment all of them but I can come back tomorrow night and push fixes to add this name on the copyright line.

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
I will work on pushing the changes to add the other author tomorrow night.

**Comment by dherrada** [comment]
@FoamyGuy Thanks for pointing that out to me!

**Review by FoamyGuy — APPROVED** [review]

**Review by FoamyGuy — APPROVED** [review]

**Pull request merged**